concourse/steps/release.py

```diff
@@ -911,6 +915,7 @@ def release_and_prepare_next_dev_cycle(
     release_commit_publishing_policy = ReleaseCommitPublishingPolicy(
         release_commit_publishing_policy
     )
+    release_notes_creation = ReleaseNotesCreation(release_notes_creation)
```

Comment:
should be index-syntax rather than call-syntax, no?

Reply:
We currently render the string-values for the enums in the corresponding mako file and instantiate the actual enum-values via the passed string.
In the interest of minimal changes I've kept it this way (c.f. l 915-917 - the used `release_commit_publishing_policy` is of type `str`)